101-recovery-services-vm-workload-backup/azuredeploy.json

```diff
@@ -82,6 +76,7 @@
     }
   },
   "variables": {
+    "vaultName": "SqlBackupDemoVault",
```

Comment:
revert this change - and add a defaultValue

Reply:
Done.